.github/workflows/linux.yml

```diff
@@ -0,0 +1,26 @@
+name: Linux
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Go 1.14.2
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.14.2
```

Comment:
IIRC, just saying "1.14" means the latest Go 1.14.x and not 1.14 exactly.

I'd rather not have to keep this in sync.